crates/typst-library/src/visualize/image.rs

```diff
@@ -32,12 +33,15 @@ use crate::text::families;
 /// Display: Image
 /// Category: visualize
 #[element(Layout, LocalName, Figurable)]
+#[scope(
+    scope.define("decode", ImageDecodeElem::func());
```

Comment:
Decode shouldn't be a separate `ImageDecodeElem`, but rather a free standing function that returns an `ImageElem`. Like how `abs`, `ceil` construct an `LrElem` in `delimited.rs`

Reply:
Yes, is is clearly better.